packages/auth/src/api/authentication/mfa.ts

```diff
@@ -79,8 +79,17 @@ export interface FinalizePhoneMfaSignInRequest {
   tenantId?: string;
 }
 
+export interface FinalizeTotpMfaSignInRequest {
+  mfaPendingCredential: string;
+  totpVerificationInfo: TotpVerificationInfo;
```

Comment:
TotpVerificationInfo has sessionInfo and verificationCode. You only need verificationCode to finalize sign in.

Reply:
Made changes